tools/scripts/AksEdgeAzureSetup/AksEdgeAzureSetup.ps1

```diff
@@ -19,6 +20,18 @@ New-Variable -Option Constant -ErrorAction SilentlyContinue -Name arcLocations -
     "westus", "northcentralus", "koreacentral", "japaneast", "eastasia", "westus3",
     "canadacentral", "eastus2euap"
 )
+
+function Set-CLILoginExperience
+{
+    $currVersion = ((az version) | ConvertFrom-Json).'azure-cli'

```

Comment:
Recommend explicitly specifying output format in case a default is set to a different format.
> az version -o json

az config get core.output
az config set core.output=table